app/Kconfig

```diff
@@ -157,6 +156,13 @@ config BT_PERIPHERAL_PREF_LATENCY
 config BT_PERIPHERAL_PREF_TIMEOUT
 	default 400
 
+if ((!ZMK_SPLIT_ROLE_CENTRAL) || (!ZMK_SPLIT)) 
```

Comment:
Maybe there is a cleaner way to write this.

Reply:
Depends on your definition of 'cleaner', but you could also do this

`if (!(ZMK_SPLIT  &&  ZMK_SPLIT_ROLE_CENTRAL))`

or just remove the parenthesis around the two terms

`if ( !ZMK_SPLIT || !ZMK_SPLIT_ROLE_CENTRAL )`

is pretty clean